tools/ci/actions/check-configure.sh

```diff
@@ -25,6 +25,15 @@
 
 set -e
 
+WANT="2.69"
+HAVE=$(sh configure -V | grep "^generated by GNU Autoconf" | awk '{print $NF}')
+if [ "$WANT" != "$HAVE" ]; then
+  echo -ne '\e[31mWrong Autoconf version: \e[0m '
+  echo -e "Have: \e[31m$HAVE\e[0m  Want: \e[32m$WANT\e[0m"
+  echo -e "Run \`make -B configure\` with autoconf-\e[32m$WANT\e[0m"
```

Comment:
It might be better here to suggest running `tools/autogen` with the name of the autoconf-2.69 command (unfortunately, on some systems it's `autoconf-2.69`, and on others it's `autoconf2.69` - the githook tries a few options at https://github.com/ocaml/ocaml/blob/trunk/tools/pre-commit-githook#L126-L129)

Reply:
I pushed change suggesting running `tools/autogen autoconf[-]$WANT`, I think it's clear enough, what do you think ?